config/networks.json

```diff
@@ -647,6 +647,24 @@
     "safeWebUrl": "https://safe.taiko.xyz/transactions/queue?safe=tko-mainnet:0xa89a87986e8ee1Ac8fDaCc5Ac91627010Ec9f772",
     "gasZipChainId": 249
   },
+  "unichain": {
+    "name": "Unichain",
+    "chainId": 130,
+    "nativeAddress": "0x0000000000000000000000000000000000000000",
+    "nativeCurrency": "ETH",
+    "wrappedNativeAddress": "0x4200000000000000000000000000000000000006",
+    "status": "active",
+    "type": "mainnet",
+    "rpcUrl": "https://mainnet.unichain.org",
+    "verificationType": "blockscout",
+    "explorerUrl": "https://unichain.blockscout.com",
+    "explorerApiUrl": "https://unichain.blockscout.com/api?",
+    "multicallAddress": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "safeApiUrl": "https://safe-transaction-unichain.safe.global/api",
+    "safeAddress": "0x428653b7537717cA9B81b453601436D576081aF6",
+    "safeWebUrl": "https://app.safe.global/transactions/queue?safe=unichain:0x428653b7537717cA9B81b453601436D576081aF6",
+    "gasZipChainId": 362
```

Comment:
could you explain gasZipChainId param? Why it's value is 362?

Reply:
@mirooon GasZip has their own internal chainId reference so when requesting gas on a specific chain, we specify their internal id which maps to whatever chain they have configured.

Reply:
pls resolve when read @mirooon